### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by djaax**
Add a simple example output for new Contrast()

### CODE REVIEW

**Comment by smockle** [comment]
Hi, thanks for the PR!

Could you please share some context for this? It seems like you’re adding this to help folks learn the instance properties of `Contrast`, but the comment references `HexColor`, which is another potentially unfamiliar class.

The instance properties of each are documented in code (e.g. [`contrast.ts#L5–10`](https://github.com/smockle/contrast/blob/d86e9df3997498c9563696d2b9ebf4c9978d5be2/src/lib/contrast.ts#L5-L10)). Code editors (e.g. VS Code) show these descriptions when properties are hovered (screenshot below):
Screen Shot 2020-05-22 at 9 11 41 AM 

Do you think it would be helpful to include a list of classes and their properties in the README? Tools (e.g. [jsdoc-to-markdown](https://github.com/jsdoc2md/jsdoc-to-markdown)) exist which could help such documentation stay in-sync with JSDoc comment changes.

**Comment by djaax** [comment]
Sure, sorry for the lack of explanation. Using `new Contrast()` I was expecting to get a contrast value returned instead of an object with the value property inside.

For first time users, I think it would indeed be helpful to see the format of the outputs I can expect from using the class. I see that it's not ideal that it refers to the HexColor class without any explanation. Maybe a more detailed output example would then be helpful? E.g.

``` 
{
    background: {
        B: { value: "5A" }
        G: { value: "24" }
        R: { value: "0E" },
        value: "0e245a"
    },
    foreground: {
        B: { value: "FF" }
        G: { value: "FF" }
        R: { value: "FF" },
        value: "FFFFFF"
    },
    value: 14.802701233126198
}
```

File: README.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ import { Contrast } from "@smockle/contrast";
 
 // Hexadecimal colors
 new Contrast("#000000", "#FFFFFF");
+  // -> { foreground: HexColor, background: HexColor, value: 21 }
```
Comment (smockle) [inline README.md:39]:
```suggestion
  // => {
  //      foreground: {
  //        value: "000000",
  //        R: { value: "00" },
  //        G: { value: "00" },
  //        B: { value: "00" },
  //      },
  //      background: {
  //        value: "FFFFFF",
  //        R: { value: "FF" },
  //        G: { value: "FF" },
  //        B: { value: "FF" },
  //      },
  //      value: 21,
  //    }
```

Comment (smockle) [reply README.md:39]: Adding a suggestion per the idea you shared in https://github.com/smockle/contrast/pull/213#issuecomment-632696945 👍

**Pull request merged**